vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -462,3 +462,23 @@ - (void)eventLog:(const std::string &)json
 }
 
 @end
+
+#import "RewardsLogStream.h"
+
+@implementation BATBraveRewardsLogger
++ (void)configure:(void(^)(int, int, NSString *, NSString *))onWrite withFlushCallback:(void(^)())flushCallback {
+    
+    UnbufferedLogger::setLoggerCallbacks([onWrite](UnbufferedLoggerData logData) {
```

Comment:
- Remove unneeded whitespace
- Should be 2 spaces instead of 4 (in the near future we will have clang-format handle this)